src/opower/utilities/exelon.py

```diff
@@ -2,16 +2,23 @@
 
 import json
 import re
+import logging
 
 import aiohttp
 
 from ..const import USER_AGENT
 from ..exceptions import InvalidAuth
 
+_LOGGER = logging.getLogger(__file__)
+
 
 class Exelon:
     """Base class for Exelon subsidiaries."""
 
+    _access_token = None
+    _subdomain = None

```

Comment:
Having the domain here is fine for now, Evergy is doing the same, but let's not add any more state in the class like access token and session. At some I want to refactor all these to be normal, non-static classes to allow adding multiple config entries in Home Assistant for the same utility but potentially different subdomain. Hopefully there shouldn't be anyone affected by this limitation, but if you add access token and session, that means people cannot have multiple config entries for the same utility and same domain which I imagine some people might be affected, e.g. with rentals in the same location.